index.js

```diff
@@ -38,23 +38,35 @@ var PROGRAMS = {
 function runVersionCommand(command, callback) {
   exec(command, function(execError, stdin, stderr) {
     var commandDescription = JSON.stringify(command);
-    if (execError && execError.code === 127) {
-      return callback(null, {
-        notfound: true,
-      });
-    }
-    else if (execError) {
-      var runError = new Error("Command failed: " + commandDescription);
-      if (stderr) {
-        runError.stderr = stderr.trim();
+
+    if (execError) {
+      if ((process.platform === 'win32') && (execError.code === 1)) {
+        if (stderr.indexOf('is not recognized')) {
```

Comment:
Relying on message strings sucks, but seems like that's our best option. Great fix!

Would it be possible to add a test that would be failing on Windows without this fix?

Reply:
I think that's a good idea, and I'm looking into this.